x-pack/plugins/security_solution/public/attack_discovery/tour/video_toast.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButton, EuiIcon, EuiToast, EuiPortal, EuiText, EuiSpacer } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React, { useCallback, useEffect } from 'react';
+import * as i18n from './translations';
+
+const VIDEO_CONTENT_HEIGHT = 160;
+const VIDEO_CONTENT_WIDTH = 250;
+const VIDEO_ID = 'BrDaDBAAvdygvemFKNAkBW';
+const VIDEO_SOURCE = `//play.vidyard.com/${VIDEO_ID}.html`;
+const VIDEO_PAGE = `https://videos.elastic.co/watch/${VIDEO_ID}`;
+
+const VideoComponent: React.FC<{ onClose: () => void }> = ({ onClose }) => {
+  const ref = React.useRef<HTMLIFrameElement>(null);
+  const [isIframeFocused, setIsIframeFocused] = React.useState(false);
+
+  const openVideoInNewTab = useCallback(() => {
+    window.open(VIDEO_PAGE, '_blank');
+  }, []);
+
+  const onIframeClick = useCallback(() => {
+    if (isIframeFocused) {
+      openVideoInNewTab();
+    }
+  }, [isIframeFocused, openVideoInNewTab]);
+
+  useEffect(() => {
+    window.addEventListener('blur', onIframeClick);
+    return () => window.removeEventListener('blur', onIframeClick);
+  }, [onIframeClick]);
+  const handleOnMouseOver = useCallback(() => {
+    setIsIframeFocused(true);
+  }, []);
+  const handleOnMouseOut = useCallback(() => {
+    setIsIframeFocused(false);
+  }, []);
+
+  return (
+    <EuiPortal>
+      <div
+        data-test-subj="attackDiscovery-tour-step-2"
+        style={{ position: 'fixed', bottom: 16, right: 16, zIndex: 9999 }}
+      >
+        <EuiToast onClose={onClose} css={{ maxWidth: VIDEO_CONTENT_WIDTH }}>
+          <div
+            css={css`
+              height: ${VIDEO_CONTENT_HEIGHT}px;
+            `}
+            onMouseOver={handleOnMouseOver}
+            onMouseOut={handleOnMouseOut}
+            onFocus={handleOnMouseOver}
+            onBlur={handleOnMouseOut}
+          >
+            <iframe
+              ref={ref}
+              allowFullScreen
+              className="vidyard_iframe"
+              frameBorder="0"
+              height="100%"
+              width="100%"
+              referrerPolicy="no-referrer"
+              sandbox="allow-scripts allow-same-origin"
+              scrolling="no"
+              // since we cannot go fullscreen, we are autoplaying and removing the controls
+              // onClick, the video will open in a new tab
```

Comment:
Clicking the 'un-mute' clicks-through and opens in a new tab:

https://github.com/elastic/kibana/assets/2946766/910a41f0-8e26-4830-9f8f-0e5e42d0b9dc

Reply:
thats the goal :D

Reply:
But then you're in a new tab with audio playing in the previous tab, so you've gotta go back to the other tab to pause before playing it in the new tab... 😅